code-of-conduct.md

```diff
@@ -0,0 +1,79 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation. This list is
+not meant to be exhaustive, but merely to enumerate some common kinds of
+behaviors.
```

Comment:
Are these common kinds of behaviors?

Perhaps we should say something like "some example of potential misbehaviors"?

Also, _sex characteristics, gender identity and expression_ and _sexual identity and orientation_ are kind of overlapping, don't they?

Reply:
Well, appearance and body size are overlapping too, not to mention "race" and "ethnicity". It's plausible that the people who wrote the original text (the Contributor Convenant) had to come to this in order to please various groups that each had their precise idea of how the wording should look like. So it's not obvious we can curate the list without exposing ourselves to dissent, trouble or some manner of public shaming.

Reply:
Sounds good, so the list is fine to me, but I'm still concerned about naming those as "common kinds of behavior".

Reply:
That makes sense. How would you reword it?

Reply:
I proposed  "some example of potential misbehaviors" above, but as a french not speeking english day to day, I'm not sure I qualify for rewording such a doc.

Reply:
"some common kinds of behaviors" -> "some examples of potential misbehaviors" is fine with me. FWIW, the term "unacceptable behavior" is used a few times throughout this document. Re-wording "some common kinds of behaviors" -> "some examples of unacceptable behavior" for consistency could also work.  

I don't feel strongly here, both of these re-wordings sound good to me. Does anyone have a preference?

Reply:
Looking at this again with fresh eyes, I have a different take on this.

It seems that CoC is trying to refer to the list of behaviors that follow (in "Our Standards"). However as it follows a list and that is a list of protected classes (not sure of the correct terminology here), it feels weird to say behaviors. Would suggest the rather minor change "This list" -> "The following list" to indicate that the list of behaviors has not yet been introduced.

Please feel free to correct me (as I may have only misread this again in a different way).

Reply:
I'm happy with changing "This list" -> "The following list"

Reply:
Sorry to revive this conversation, but if you want a somewhat broader definition of the characteristics here, you may list/mention no [_ad hominem_](https://en.wikipedia.org/wiki/Ad_hominem) argumentation is tolerated.